packages/react-magma-dom/src/components/NativeSelect/NativeSelect.test.js

```diff
@@ -34,4 +26,64 @@ describe('NativeSelect', () => {
       return expect(result).toHaveNoViolations();
     });
   });
+
+  it('should render a disabled select', () => {
+    const testId = 'test-id';
+    const { getByTestId } = render(
+      <NativeSelect disabled testId={testId}></NativeSelect>
+    );
+    const nativeselect = getByTestId(testId);
+    expect(nativeselect).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+  });
+
+  it('should render a disabled inverse select', () => {
+    const testId = 'test-id';
+    const { getByTestId } = render(
+      <NativeSelect disabled isInverse testId={testId}></NativeSelect>
+    );
+    const nativeselect = getByTestId(testId);
+    expect(nativeselect).toHaveStyleRule(
+      'color',
+      transparentize(0.6, magma.colors.neutral100)
+    );
+  });
+
+  it('should render an error state', () => {
+    const testId = 'test-id';
+    const { getByTestId, getByText } = render(
+      <NativeSelect
+        errorMessage="This is an error"
+        testId={testId}
+      ></NativeSelect>
+    );
+
+    const nativeselect = getByTestId(testId).parentElement;
+    expect(nativeselect).toHaveStyleRule('border-color', magma.colors.danger);
+
+    const errorMessage = 'This is an error';
+    expect(getByText(errorMessage)).toBeInTheDocument();
+  });
+
+  it('should render an inverse error state', () => {
+    const testId = 'test-id';
+    const { getByTestId, getByText } = render(
+      <NativeSelect
+        errorMessage="This is an error"
+        isInverse
+        testId={testId}
+      ></NativeSelect>
+    );
+
+    const nativeselect = getByTestId(testId).parentElement;
+    expect(nativeselect).toHaveStyleRule(
+      'border-color',
+      magma.colors.danger200
+    );
+
+    const errorMessage = 'This is an error';
+    expect(getByText(errorMessage)).toBeInTheDocument();
```

Comment:
**This is style feedback and not required changes. Take it or leave it**

1. I would suggest declaring the `errorMessage` variable at the top of the test so you can use it in multiple places instead of retyping `'This is an error'` twice. This way, if we wanted to change what that message says we would only need to do it once.

2. Typically if I'm only going to use a value once, I don't create a variable (unless it would really help for readability). I would probably just have this expect in a single line `expect(getByTestId(testId).parentElement).toHaveStyleRule(.......);`

```
    const testId = 'test-id';
    const errorMessage = 'This is an error';

    const { getByTestId, getByText } = render(
      <NativeSelect
        errorMessage={errorMessage}
        isInverse
        testId={testId}
      ></NativeSelect>
    );

    expect(getByTestId(testId).parentElement).toHaveStyleRule(
      'border-color',
      magma.colors.danger200
    );

    expect(getByText(errorMessage)).toBeInTheDocument();
```